teams/example/backend.tf

```diff
@@ -6,8 +6,8 @@ terraform {
     acl                  = "bucket-owner-full-control"
     bucket               = "modernisation-platform-terraform-state"
     encrypt              = true
-    key                  = "imagebuilder-team1.tfstate"
+    key                  = "imagebuilder-team1.tfstate" # replace team1 with your team name when copying this file
```

Comment:
As discussed, could we leave this line entirely commented out, to have it fail until team engineer uncomments it?